portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,37 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself!
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function genRandomFact() {
+  const facts = [
+    'I love spicey food, but I dislike eating peppers!',
+    'I play the flute!',
+    'I have been taking lots of walks during quarantine',
+    'I am 19 years old!',
+  ];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
 }
+
+/**
+ * Implements Sticky Nav Bar
+ */
+document.addEventListener('DOMContentLoaded', function() {
+  window.onscroll = function() {stickyNav()};
+  var navbar = document.getElementById("nav");
+  var offsetPos = navbar.offsetTop;
+  
+  // Add the sticky class to nav when scroll, remove when not
+  function stickyNav() {
+    if (window.pageYOffset >= offsetPos) {
+      navbar.classList.add("sticky")
+    } else {
```

Comment:
If you are only using this once then I would inline to line 24, otherwise I would move the function declaration outside the event listener!

Reply:
yes, use only once. Moved it to inline with 24 :)